UI/MapStatisticsView.cs

```diff
@@ -69,7 +74,66 @@ public string MapStatisticsString()
-      stats += $"Total # Rooms with Objects: {MapStatistics.NumberOfRoomsWithObjects}{Environment.NewLine}";
+      stats += $"Total # Rooms with Objects: {MapStatistics.NumberOfRoomsWithObjects}, {MapStatistics.NumberOfRoomsWithXObjects(1, false)} with 1, " +
+        $"{MapStatistics.NumberOfRoomsWithXObjects(2, false)} with 2, {MapStatistics.NumberOfRoomsWithXObjects(3, true)} with 3+{Environment.NewLine}";
+
+      stats += $"{Environment.NewLine}";
+      stats += $"Total # of Rooms with subtitles: {MapStatistics.NumberOfRoomsWithSubtitles}{Environment.NewLine}";
+      stats += $"Total # of Rooms with descriptions: {MapStatistics.NumberOfDescribedRooms}{Environment.NewLine}";
+
+      stats += $"{Environment.NewLine}{new String('=', 30)}Odd stuff below here{Environment.NewLine}";
+
+      if (MapStatistics.NumberOfRegions > 0)
+      {
+        stats += $"{Environment.NewLine}";
+        foreach (var region in Settings.Regions.OrderBy(p=>p.RegionName).Where(p=>p.RegionName != Trizbort.Region.DefaultRegion))
+        {
+          if (MapStatistics.NumberOfRoomsInRegion(region.RegionName) == 0)
+            stats += $"{region.RegionName} has no rooms.";
+          else if (MapStatistics.NumberOfRoomsInRegion(region.RegionName) == 1)
+            stats += $"Only room in {region.RegionName}: ";
+          else
+            stats += $"List of rooms in {region.RegionName}: ";
+
+          var roomsInRegion = Project.Current.Elements.OfType<Room>().ToArray().OrderBy(p=>p.Name).Where(p=>p.Region == region.RegionName);
+          stats += string.Join(", ", roomsInRegion.Select(x => x.Name).ToArray()) + Environment.NewLine;
+        }
+        stats += $"{Environment.NewLine}";
+      }
+
+      if (MapStatistics.NumberOfRegions > 1)
+      {
+        stats += $"{Environment.NewLine}Region Links:{Environment.NewLine}";
+        var allRegions = Settings.Regions.OrderBy(p => p.RegionName).Where(p => p.RegionName != Trizbort.Region.DefaultRegion);
+        foreach (var region1 in allRegions)
+        {
+          var linkedRegions = string.Join(", ", Settings.Regions.OrderBy(p => p.RegionName).ToArray().Where(p => MapStatistics.RegionsLinked(region1, p)).Select(x => x.RegionName));
+
+          stats += $"{Environment.NewLine}";
+
+          if (linkedRegions == $"")
+            stats += $"{region1.RegionName} is not linked to any other regions.";
+          else
+            stats += $"{region1.RegionName} is linked to {linkedRegions}.";
+
+        }
+        stats += $"{Environment.NewLine}";
+      }
+
+      stats += string.Format("{0} up-down connection{1}{2}", MapStatistics.UpDown, plur(MapStatistics.UpDown), Environment.NewLine);
+      stats += string.Format("{0} in-out connection{1}{2}", MapStatistics.InOut, plur(MapStatistics.InOut), Environment.NewLine);
+      stats += string.Format("{0} miscellaneous {1}connection{2}", MapStatistics.MiscConnect, plur(MapStatistics.MiscConnect), Environment.NewLine);
```

Comment:
I think "custom" would be better, because that is what is in the connection editing GUI.